### PR TITLE
Re-fix types in Vue demos

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

**Review by mpreyskurantov — DISMISSED** [review]

File: JSDemos/Demos/DataGrid/CustomEditors/Vue/EmployeeDropDownBoxComponent.vue [hunk 1]
```diff
@@ -57,7 +56,7 @@ const props = defineProps<{
 
 const currentValue = ref(props.value);
 const dropDownBoxRef = ref<DxDropDownBox | null>(null);
-const dropDownOptions: DxPopupTypes.DropDownOptions = { width: 500 };
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/DataGrid/CustomEditors/Vue/EmployeeDropDownBoxComponent.vue:60]: Why did you remove the type?

Comment (GoodDayForSurf) [reply JSDemos/Demos/DataGrid/CustomEditors/Vue/EmployeeDropDownBoxComponent.vue:60]: yes, we need this type here

**Review by dxvladislavvolkov — APPROVED** [review]
